### ISSUE TITLE
Settings should have "OK" or "apply" buttons [minor]

### ISSUE THREAD

**Issue by szopeno** [olivierkes/manuskript#78]
The Settings dialog boxes should have "ok" or "apply" button. I was playing the options  and I was not sure whether the changes have immediate effect or not.

**Comment by olivierkes** [comment]
Yep the changes have immediate effect. An "apply" button would be misleading, and clutter the interface.
Any other ways to show the changes have been saved? A status text that disapears after a few seconds?
